pg_store_plans.c

```diff
@@ -1579,6 +1579,12 @@ pg_store_plans_internal(FunctionCallInfo fcinfo,
 			else
 				pstr = SHMEM_PLAN_PTR(entry);
 
+			if (pstr == NULL)
+			{
+				values[i++] = CStringGetTextDatum("<invalid plan>");
```

Comment:
I think it would be better to set the value as null rather than adding another hardcoded value that users will have to explicitly check.

Also, since the code is not that complicated it would be better to avoid a `goto`and just nest the instructions.